e2e/tests/transfer/base_test.go

```diff
@@ -467,180 +499,6 @@ func (s *TransferTestSuite) TestMsgTransfer_Timeout_Nonincentivized() {
 	})
 }
 
-// TestSendEnabledParam tests changing ics20 SendEnabled parameter
```

Comment:
moved into its own test suite since it messes with chain params and breaks other tests.